web/AdminLteAsset.php

```diff
@@ -15,7 +15,8 @@ class AdminLteAsset extends AssetBundle
         'css/skins/_all-skins.min.css'
     ];
     public $js = [
-        'js/app.min.js'
+        'js/app.min.js',
+        'js/demo.js',
```

Comment:
then we need to make another asset just for the demo or include that part on the `layout/main.php` file. Otherwise demo.js will load on every page, including the ones that doesn't follow the demo structure, some times colliding with the functionality the developer wants to implement.